gulp-core-build-typescript/external-api-types/whatwg-fetch/index.d.ts

```diff
@@ -0,0 +1,111 @@
+// Type definitions for fetch API
```

Comment:
At the top of this file, we should have a note like this:

"Typings based on DefinitelyTyped definitions from whatwg-fetch@1.2.3"

But delete all other source code comments, since we will be authoring our own original content.